src/api/stats/fantom/getStakeSteakPrices.ts

```diff
@@ -0,0 +1,166 @@
+import pools from '../../../data/fantom/stakesteakLpPools.json';
+import { fantomWeb3 as web3 } from '../../../utils/web3';
+import ERC20 from '../../../abis/ERC20.json';
+import BigNumber from 'bignumber.js';
+import Web3 from 'web3';
+import { MULTICALL_V3 } from '../../../utils/web3Helpers';
+import { ContractCallContext, Multicall, CallReturnContext } from 'ethereum-multicall';
+import { ChainId } from '../../../../packages/address-book/types/chainid';
+import { groupBy } from 'lodash';
+
+type PoolType = typeof pools[number];
+
+type LpBreakdown = {
+  price: number;
+  tokens: string[];
+  balances: string[];
+  totalSupply: string;
+};
+
+export async function getStakeSteakPrices(
+  tokenPrices: Record<string, number>
+): Promise<Record<string, LpBreakdown>> {
+  const multicallAddress = MULTICALL_V3[ChainId.fantom];
+  return getPoolsData(web3, multicallAddress, pools, tokenPrices);
+}
+
+async function getPoolsData(
+  web3: Web3,
+  multicallAddress: string,
+  pools: PoolType[],
+  tokenPrices: Record<string, number>
+): Promise<Record<string, LpBreakdown>> {
+  const multicall = new Multicall({
+    web3Instance: web3,
+    tryAggregate: true,
+    multicallCustomContractAddress: multicallAddress,
+  });
+
+  const calls: Erc20Call[] = pools.reduce((acc: Erc20Call[], pool: PoolType) => {
```

Comment:
this is quite a code for one lp pool!

unfortunately it is also not correct. 
this LP is something else, it is not a Solidly one.
normally you can just fetch `reserves` from LP, could be a call to 1 lp contract reserves + supply, no need `balanceOf` as it is synced to reserves on every action.

but in this LP balances are not in sync with reserves, 47k USDC that LP holds are just not counted, actual reserves are 750 fUSD and 272 USDC, everything else is lost.
so with totalSupply of 1000, lp price would be ~1, looks correct.

i would suggest to just delete this LP as its broken anyway, nobody swaps through it either. vault is retired long time ago we can delete it from app to avoid error in logs